src/aspire/classification/averager2d.py

```diff
@@ -228,6 +227,27 @@ def _innerloop(i):
         # Now we convert the averaged images from Basis to Cartesian.
         return ArrayImageSource(self.composite_basis.evaluate(b_avgs))
 
+    def _shift_search_grid(self, L, radius, roll_zero=False):
+        """
+        Returns two vectors representing the grid points in the disc <= self.radius.
```

Comment:
"Returns two 1-D arrays representing the X and Y grid points in the defined shift search space (`disc <= self.radius`)?"

Reply:
Yeah, that sounds better to me. Thanks